app/js/components/orderEntry/SortAndFilter.jsx

```diff
@@ -14,6 +14,7 @@ export const SortAndFilter = props => (
         <option value="all">All</option>
         <option value="drugorder">Drug Orders</option>
         <option value="testorder">Test Orders</option>
+        <option value="orderset">Order from Sets</option>
```

Comment:
`Order from Sets` is not a new type of order, it is just a way of grouping orders together. So I don't think there should be a functionality like sorting with "Order from Sets".

Reply:
Ok will remove it